tap_marketo/sync.py

```diff
@@ -139,10 +139,14 @@ def stream_rows(client, stream_type, export_id):
         resp = client.stream_export(stream_type, export_id)
         for chunk in resp.iter_content(chunk_size=CHUNK_SIZE_BYTES, decode_unicode=True):
             if chunk:
+                # Replace CR and CRLF
+                chunk = chunk.replace('\r', '')
+                chunk = chunk.replace('\r\n', '')
```

Comment:
We should not replace `\r\n` since that's a valid Windows return sequence. Removing the `\r` should suffice while still maintaining the behavior of "this is a newline"